### ISSUE TITLE
Can't add multiple Tiles Collision Shapes with different physics_layer

### ISSUE THREAD

**Issue by Lertsenem** [Kiamo2/YATI#37]
**Describe the bug**
I'm trying to setup a Tile with two different collision rectangles. One is on the `physics_layer` 0, the other should be on the `physics_layer` 1 (which has different collisions layer, etc).
When importing my map, only the first rectangle (`physics_layer = 0`) shows up. An error is raised in the console output :

```
scene/resources/tile_set.cpp:5431 - Index p_polygon_index = 1 is out of bounds (physics[p_layer_id].polygons.size() = 1).
```

The map is still imported, but there is no collision rectangle for the physics_layer 1

**To Reproduce**
1. Create a map in Tiled, edit the tilemap to add collision shapes to some tile
2. Create a basic rectangle without any property, this will be the collision rectangle for physics_layer 0
3. Create another rectangle, this time adding the int property `physics_layer` and setting it to 1. This rectangle should be imported as a collision rectangle for physics_layer 1
4. Go to Godot and import that map using YATI, you should get the error

```
scene/resources/tile_set.cpp:5431 - Index p_polygon_index = 1 is out of bounds (physics[p_layer_id].polygons.size() = 1).
```

5. Additionally, you can check that the tile you worked on has no collision shape/polygon on physics_layer 1.

**Expected behavior**
The collision rectangle/polygon should be imported on physics_layer 1 for that tile.

**Screenshots**
Screenshots of the Tiled setup :
![shape0](https://github.com/Kiamo2/YATI/assets/1083508/585ee5c8-381d-41a1-9dfe-369045ca699c)
![shape1](https://github.com/Kiamo2/YATI/assets/1083508/4437381a-c2d4-40ad-aa6d-92e4753c088b)

Screenshot of the resulting import in Godot :
![gshape](https://github.com/Kiamo2/YATI/assets/1083508/46eb8e90-21d0-4d07-acc7-c8b7887ba3b4)

**Desktop**
 - OS: Linux - Ubuntu 20.04 (LTS)
 - Version: 1.6.3 - GDScript
 - Godot Version: 4.2.1 stable
 - Tiled Version: 1.10.2

**Example map (zipped)**
Here's a small example.

- The white tile has no collision rectangle
- The black tile is setup with only one collision rectangle on physics_layer 0 and is imported properly
- The red tile is setup with only one collision rectangle on physics_layer 1 and is imported properly
- The blue tile is setup with collisions rectangles on both physics_layer 0 and 1 and only the collision rectangle for physics_layer 0 is imported.

[mini.zip](https://github.com/Kiamo2/YATI/files/14064423/mini.zip)

**Comment by Kiamo2** [comment]
That's exactly what I did :-)
I will release the fix tomorrow together with another change.

**Comment by Kiamo2** [comment]
Fixed in released version 1.6.5
